app/grandchallenge/evaluation/forms.py

```diff
@@ -633,11 +664,11 @@ def check_submission_limit_avoidance(self, *, creator):
     def save(self, *args, **kwargs):
         if self._phase.submission_kind == SubmissionKindChoices.ALGORITHM:
             self.instance.algorithm_requires_gpu_type = self.cleaned_data[
-                "algorithm_image"
-            ].algorithm.job_requires_gpu_type
+                "algorithm"
```

Comment:
Why the change here to look at the algorithm rather than the algorithm image?

Reply:
It's completely unrelated to this PR, sorry. I stumbled upon this and wondered why we look at the algorithm image to get the algorithm when we can get the algorithm directly. I changed it to test if something would break.